applications/Chat/evaluate/metric.py

```diff
@@ -0,0 +1,152 @@
+import jieba
+from nltk.translate.bleu_score import sentence_bleu
+from rouge_chinese import Rouge
```

Comment:
If those are external packages, you should add the packages in the installation process.